src/test/java/com/Model/ControllerTest/EventControllerTest.java

```diff
@@ -49,8 +49,8 @@ public void testPostPatients () throws Exception{
         mockMvc.perform(post("/patientinfo").content(jsonString)
```

Comment:
We need a lot more tests here! For all cases of interactions with the various REST endpoints.